Samples/HelloWorld/OrleansClient/Program.cs

```diff
@@ -26,29 +45,22 @@ static int Main(string[] args)
                     Console.WriteLine("Client successfully connect to silo host");
                     break;
                 }
-                catch (SiloUnavailableException e)
+                catch (SiloUnavailableException)
                 {
-                    attemp++;
-                    if (attemp > initializeAttemptsBeforeFailing)
+                    attempt++;
+                    Console.WriteLine($"Attempt {attempt} of {initializeAttemptsBeforeFailing} failed to initialize the Orleans client.");
```

Comment:
we should state here client failed connect here because silo host hasn't start yet, otherwise it confuse user see clients keep failing connecting.